suzieq/engines/pandas/ospf.py

```diff
@@ -425,12 +425,15 @@ def _get_peernames(self, df: pd.DataFrame, columns: List[str],
         if newdf.empty:
             return df
 
-        lldp_df = self._get_table_sqobj('lldp').get(
+        addr_df = self._get_table_sqobj('address').get(
```

Comment:
Why do we changed from lldp to address here?

Reply:
We need to handle non-p2p interfaces (think Vlan) which doesn't work with lldp.